plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -27,6 +31,8 @@ export type SortOption = "ID" | "STATUS" | "LAST_RUN";
 
 export type SortDirection = "ASC" | "DESC";
 
+export type JobRunStatus = "FAILED" | "NOT_AVAILABLE" | "SUCCESS";
```

Comment:
The expected states, and the states on JobStates are "Failed", "N/A" and "Success". If we decide to change it here, we have to fix it everywhere, otherwise, it will be just a huge source of confusion.